builder/bundles.go

```diff
@@ -342,8 +342,16 @@ func queryRpmFullPath(packageCmd []string, pkgName string, repo string) (string,
 	if pURL.Scheme == "file" { // obtain the full path without the url scheme
 		pURL.Scheme = ""
 		rpmFullPath = pURL.String()
+		// If repo cache dir has not been guessed before, guess it just one time.
+		// This is an attempt to improve performance in case the rpms are not stored directly within the repo baseurl.
+		if !repos[repo].guessCache {
```

Comment:
I don't think `guessCache` is necessary right now. We probably will only call `queryRpmFullPath` when guessCache was a miss, so we could update it.

Alternatively, we could maintain a list of all the sub-directories with packages for a given repo. The `stat` commands could try all known sub-directories.

Reply:
The reason guessCache is used is so that the guess is just done once in case there are multiple sub-dirs within the baseurl.

But if we make the change to make cacheDir as a list of directories, guessCache will not be required.

Reply:
@jwakre Updated PR based on your suggestion
`Alternatively, we could maintain a list of all the sub-directories with packages for a given repo. The stat commands could try all known sub-directories.`